pkg/synopsysctl/cmd_update.go

```diff
@@ -391,11 +391,8 @@ var updateBlackDuckCmd = &cobra.Command{
 
 func runPostgresMigration(blackDuckName string, blackDuckNamespace string, oldVersion string, newVersion string, helmValuesMap map[string]interface{}) error {
 	// If this instance is not using the PG container, do nothing and return
-	isExternal := util.GetHelmValueFromMap(helmValuesMap, []string{"postgres", "isExternal"})
-	if isExternal == nil {
-		return fmt.Errorf("postgres.isExternal must be specified")
-	}
-	if isExternal.(bool) {
+	isExternal := util.GetHelmValueFromMap(helmValuesMap, []string{"postgres", "host"}) != nil
```

Comment:
Deleted code was added in 3.0.0, but since we don't have postgres.isExternal option in synopsysctl, it throws error. Therefore i use the host to check if postgres is external or not